server/helpers/reaction.js

```diff
@@ -0,0 +1,23 @@
+import {
+  LIKE_REACTION,
+  DISLIKE_REACTION,
+  LIKE_COUNT,
+  DISLIKE_COUNT
+} from "@helpers/constants";
+
+export const typeOfReaction = reaction => {
+  switch (reaction) {
+    case true:
```

Comment:
I do not believe this abstraction is saving us from writing less code.  Why are you repeating the variable LIKE_COUNT

Reply:
@obasajujoshua31